src/com/Jacksonnn/DCCore/BannedWords/BannedWordsCommand.java

```diff
@@ -20,29 +20,29 @@ public boolean onCommand(CommandSender sender, Command cmd, String commandLabel,
         if (sender.hasPermission("DCCore.bannedwords")) {
             if (args.length == 0) {
                 sender.sendMessage(GeneralMethods.prefix + "Banned Words Commands:");
-                sender.sendMessage(ChatColor.YELLOW + "/bannedwords list");
-                sender.sendMessage(ChatColor.YELLOW + "/bannedwords add <word>");
-                sender.sendMessage(ChatColor.YELLOW + "/bannedwords remove <word>");
+                sender.sendMessage(ChatColor.of("#E5D900") + "/bannedwords list");
+                sender.sendMessage(ChatColor.of("#E5D900") + "/bannedwords add <word>");
+                sender.sendMessage(ChatColor.of("#E5D900") + "/bannedwords remove <word>");
```

Comment:
It would be nice if we could get a static reference of ChatColor.of("#e5d900"); in GeneralMethods, similar to how we have GeneralMethods.accentColor